src/block/Lava.php

```diff
@@ -94,7 +94,8 @@ public function onEntityInside(Entity $entity) : bool{
 		$ev = new EntityDamageByBlockEvent($this, $entity, EntityDamageEvent::CAUSE_LAVA, 4);
 		$entity->attack($ev);
 
-		$ev = new EntityCombustByBlockEvent($this, $entity, 15);
+		//in java burns entities for 15 seconds - seems to be a parity issue in bedrock
```

Comment:
have you verified this?

Reply:
Yes, minecraft java:

https://user-images.githubusercontent.com/57236932/179421901-d7e5a7ea-1d11-4504-b39a-f63482351eaf.mp4